addons/binding/org.openhab.binding.airquality/README.md

```diff
@@ -0,0 +1,128 @@
+|-----------|------------------------------------------------------------------------- |
+| apikey    | Data-platform token to access the AQIcn.org service. Mandatory. |
+| location  | Geo coordinates to be considered by the service. Mandatory. |
+| refresh   | Refresh interval in minutes. Optional, the default value is 60 minutes.  |
+
+For the location parameter, the following syntaxes are allowed (comma or semicolon):
+
+`37.8;-122.4`
+`37.8255,-122.456`
+
+
+## Channels
+
+The AirQuality information that is retrieved is available as these channels:
+
+
+| Channel ID | Item Type    | Description              |
+|------------|--------------|------------------------- |
+| aqiLevel | Number | Air Quality Index |
+| aqiDescription | String | AQI Description |
+| locationName | String | Nearest measuring station location |
+| locationUrl | String | Direct URL to the station results |
+| stationId | Number | Unique identifier of measuring station |
+| pm25 | Number | Fine particles pollution level (PM2.5) |
+| pm10 | Number | Coarse dust particles pollution level (PM10) |
+| o3 | Number | Ozone level (O3) |
+| no2 | Number | Nitrogen Dioxide level (NO2) |
+| co | Number | Carbon monoxide level (CO) |
+| observationTime | DateTime | Observation date and time |
+| temperature | Number | Temperature in Celsius degrees |
+| pressure | Number | Pressure level |
+| humidity | Number | Humidity level |
+
+`AQI Description` item provides a human-readable output that can be interpreted e.g. by MAP transformation.
+
+*Note that channels like* `pm25`, `pm10`, `o3`, `no2`, `co` *can sometimes return* `UNDEF` *value due to the fact that some stations don't provide measurements for them.*
+
+
+## Full Example
+
+airquality.map:
+
+```
+-=-
+UNDEF=No data
+NULL=-No data
+NO_DATA=No data
+GOOD=Good
+MODERATE=Moderate
+UNHEALTHY_FOR_SENSITIVE=Unhealthy for sensitive groups
+UNHEALTHY=Unhealthy
+VERY_UNHEALTHY=Very unhealthy
+HAZARDOUS=Hazardous
+```
+
+airquality.things:
+
+```
+airquality:aqi:home "AirQuality" @ "Krakow" [ apikey="XXXXXXXXXXXX", location="50.06465,19.94498", refresh=60 ]
+airquality:aqi:warsaw "AirQuality in Warsaw" [ apikey="XXXXXXXXXXXX", location="52.22,21.01", refresh=60 ]
+```
+
+airquality.items:
+
+```
+Group AirQuality <flow>
+
+Number   Aqi_Level           "Air Quality Index" <flow> (AirQuality) { channel="airquality:aqi:home:aqiLevel" }
+String   Aqi_Description     "AQI Level [MAP(airquality.map):%s]" <flow> (AirQuality) { channel="airquality:aqi:home:aqiDescription" }
+
+Number   Aqi_Pm25            "PM25 Level" <line> (AirQuality) { channel="airquality:aqi:home:pm25" }
+Number   Aqi_Pm10            "PM10 Level" <line> (AirQuality) { channel="airquality:aqi:home:pm10" }
+Number   Aqi_O3              "O3 Level" <line> (AirQuality) { channel="airquality:aqi:home:o3" }
+Number   Aqi_No2             "NO2 Level" <line> (AirQuality) { channel="airquality:aqi:home:no2" }
```

Comment:
Same goes for Pm25 and pm10. Too bad I haven't been able to locate a subscript dot (for subscript 2.5).